app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -8,229 +8,28 @@
     %p.govuk-body-l
       - if @vacancy.publish_on.today?
         = t('jobs.review')
-      - else 
+      - else
         = t('jobs.review_future')
     = render 'hiring_staff/vacancies/error_messages', errors: @vacancy.errors
 
 .vacancy.govuk-grid-row
   .govuk-grid-column-two-thirds
+    = render 'hiring_staff/vacancies/review_vacancy_sections/review_job_specification'
+    - if UploadDocumentsFeature.enabled?
+      = render 'hiring_staff/vacancies/review_vacancy_sections/review_supporting_documents'
+    - else
+      = render 'hiring_staff/vacancies/review_vacancy_sections/review_candidate_specification'
```

Comment:
When Upload Docs is enabled this wouldn't show the review candidate specification stuff and there would be existing live vacancies in the service with no documents uploaded to them

Reply:
I guess we should show both until the transition to the new journey is complete for all users and all vacancies

Reply:
@dushan-madetech, is the review page seen (a) only at the end of the process of creating a new vacancy, or (b) are there other times it's seen? If (a), new vacancies created after the feature flag is switched on will not have candidate spec', so this is fine, but if (b) then yes, we need to cater for legacy vacancies.

Reply:
Actually, even if (a), there may be some draft vacancies created before the feature flag is switched on, which have a candidate specification.